pkgs/development/ocaml-modules/base64/default.nix

```
@@ -2,13 +2,13 @@
 
 buildDunePackage rec {
   pname = "base64";
-  version = "3.4.0";
+  version = "3.5.0";
 
   useDune2 = true;
 
   src = fetchurl {
     url = "https://github.com/mirage/ocaml-base64/releases/download/v${version}/base64-v${version}.tbz";
-    sha256 = "0d0n5gd4nkdsz14jnxq13f1f7rzxmndg5xql039a8wfppmazd70w";
+    sha256 = "sha256-WJ3pwAV46/54QZismBjTWGxHSyMWts0+HEbMsfYq46Q=";
   };
 
   buildInputs = [ bos dune-configurator ];
```

`dune-configurator` is no longer needed.

```suggestion
  buildInputs = [ bos ];
```